google/cloud/forseti/scanner/scanners/enabled_apis_scanner.py

```diff
@@ -15,7 +15,6 @@
 """Scanner for Enabled APIs."""
 
 import json
-import sys
 
 from google.cloud.forseti.common.gcp_type.project import Project
 from google.cloud.forseti.common.util import logger
```

Comment:
Hi Brandon, you will have to import like this.  Thanks.
```
from google.cloud.forseti.common.util import errors as util_errors

raise util_errors.NoDataError

except util_errors.NoDataError
```

https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/google/cloud/forseti/common/util/errors.py#L42-L44

Reply:
Just caught that.  New build coming.  Sorry about that.